src/components/BookAChat/index.tsx

```diff
@@ -320,15 +320,14 @@ const BookAChat = ({ mentor }: BookAChatProps) => {
             </Text>
           </div>
           <div className="h-6"></div>
-
           <TextArea
             value={chatRequestMessage}
             onChange={(e: any) => {
               const target = e.target as HTMLTextAreaElement;
               setChatRequestMessage(target.value);
             }}
             className="p-2 w-96"
-            placeholder="Optional message"
+            placeholder="Suggest a location (Zoom link, in person address, etc) and leave an optional message."
```

Comment:
etc -> etc.

Consider: "Provide any information that could help with scheduling a meeting (preferred location, Zoom link, a custom message, etc.)"

(Your judgment in which is better)

Reply:
I like the way it is now just because it prompts them to do both separately instead of combining the prompts into one.